fluent/client/ws_client.go

```diff
@@ -114,12 +112,12 @@ func (wcf *DefaultWSConnectionFactory) New() (ext.Conn, error) {
 	}
 
 	conn, resp, err := dialer.Dial(wcf.URL, header)
+
 	if resp != nil && resp.Body != nil {
 		bodyBytes, readErr := io.ReadAll(resp.Body)
```

Comment:
Add `respo.Body.Close()` here like so:

```
if resp != nil && resp.Body != nil {
        defer resp.Body.Close()
	bodyBytes, readErr := io.ReadAll(resp.Body)
}
```

Reply:
actually initially i tried defer but travis was giving some error. 
Since i have updated `go` version i will add and recheck